<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Decode</class>
 <widget class="QWidget" name="Decode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>620</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1050</width>
    <height>620</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frBg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frBg{
background-image:url(:/main/img/bg.jpg);
}
QFrame#frWhite{
background-color:rgb(255,255,255,230);
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frWhite">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>850</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
	font-family:STXihei, &quot;华文细黑&quot;, &quot;Microsoft YaHei&quot;, &quot;微软雅黑&quot;;
font-size:20px;
}
QPushButton{
border-radius:15px;
background:rgb(237,200,199)

}
QPushButton:hover{
background:rgb(255,208,204);
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnDecodeDe">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>440</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDecodeCancel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>440</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDecodePath">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>101</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>导入视频</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditDecodePath">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>450</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDecodeSelect">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>170</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditOutput">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>270</y>
       <width>450</width>
       <height>40</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDecodeSelect2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>270</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDecodePath_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>151</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>二进制导出目录</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
